ansible/roles/rancher/tasks/password.yml

```diff
@@ -25,4 +24,4 @@
       newPassword: "{{ rancher_password }}"
     validate_certs: no
     return_content: yes
-  when: login is success
\ No newline at end of file
+  when: login.status_code == 201
```

Comment:
I believe `201` is included in "success"

Reply:
Ah cool, yeah it matches the status_code list if I'm reading it correctly
https://docs.ansible.com/ansible/latest/modules/uri_module.html#parameter-status_code